app/controllers/admin/routes.py

```diff
@@ -95,22 +95,26 @@ def createEvent(templateid, programid=None):
     # Try to save the form
     if request.method == "POST":
         try:
-            saveSuccess, validationErrorMessage = attemptSaveEvent(eventData, attachmentFiles)
-            createLog(f"Created event: {eventData['name']}, which had a start date of {datetime.strftime(eventData['startDate'], '%m/%d/%Y')}")
-
+            savedEvents, validationErrorMessage = attemptSaveEvent(eventData, attachmentFiles)
         except Exception as e:
             print("Error saving event:", e)
-            saveSuccess = False
+            savedEvents = False
             validationErrorMessage = "Unknown Error Saving Event. Please try again"
 
-        if saveSuccess:
+        if savedEvents:
+
             noun = (eventData['isRecurring'] == 'on' and "Events" or "Event") # pluralize
             flash(f"{noun} successfully created!", 'success')
-            eventId = Event.select(fn.MAX(Event.id)).scalar() #gets the last created event
-            if eventData['isRecurring']: 
-                lastEventInSeries = Event.get(Event.id == eventId)
-                eventId = Event.get(Event.recurringId == lastEventInSeries.recurringId)
-            return redirect(url_for("admin.eventDisplay", eventId = eventId))
+
+            if not program:
+                createLog(f"Created a non-program event: <a href=\"{url_for('admin.eventDisplay', eventId = savedEvents[0].id)}\">{savedEvents[0].name}</a>, with a start date of {datetime.strftime(eventData['startDate'], '%m/%d/%Y')}.")
+            else:
+                if len(savedEvents) > 1:
+                    createLog(f"Created a recurring event, <a href=\"{url_for('admin.eventDisplay', eventId = savedEvents[0].id)}\">{savedEvents[0].name}</a>, for {program.programName}, with a start date of {datetime.strftime(eventData['startDate'], '%m/%d/%Y')}. The last event in the series will be on {datetime.strftime(savedEvents[-1].startDate, '%m/%d/%Y')}.")
```

Comment:
Add colons when creating and deleting events to be consistent. This is a good example of where you use a colon "Created a non-program event:"